Environment/carbon-intensity.15m.js

```diff
@@ -0,0 +1,355 @@
+#!/usr/bin/env -S PATH="${PATH}:/opt/homebrew/bin:/usr/local/bin" node
+
+// Metadata
+// <xbar.title>Carbon Intensity</xbar.title>
+// <xbar.version>v1.3</xbar.version>
+// <xbar.author>Jason Jones</xbar.author>
+// <xbar.author.github>jasonm-jones</xbar.author.github>
+// <xbar.desc>Shows real-time carbon intensity and grid cleanliness to help you minimize your carbon footprint by running energy-intensive tasks at cleaner times.</xbar.desc>
+// <xbar.image>https://github.com/jasonm-jones/carbon-intensity-xbar/raw/main/carbon-intensity-screenshot-v1.1.png</xbar.image>
```

Comment:
Image URL leads to a 404 page. Could you make sure it is hosted somewhere publicly please?